plugins/modules/pipx.py

```diff
@@ -54,11 +54,16 @@
   name:
     type: str
     description:
-      - The name of the application. In C(pipx) documentation it is also referred to as the name of the virtual environment
-        where the application will be installed.
+      - The name of the application and also the name of the Python package being installed.
+      - In C(pipx) documentation it is also referred to as the name of the virtual environment
+        where the application is installed.
       - If O(name) is a simple package name without version specifiers, then that name is used as the Python package name
         to be installed.
-      - Use O(source) for passing package specifications or installing from URLs or directories.
+      - Starting in community.general 10.7.0, you can use package specifiers when O(state=present) or O(state=install). For example, O(name=tox<4.0.0).
```

Comment:
hi @felixfontein I am not expecting this to make the cut into 10.6.0 so I marked it for 10.7.0. If you review and find it  ~~terrifically awesome~~ good enough to go sooner rather than later, I'm happy to jump in and fix the version in the doc here before the release. Please let me know.

Reply:
... and that ship has sailed :-)